index.html

```diff
@@ -752,41 +737,22 @@ <h2>Attributes</h2>
               be fired any time the <code><a>RTCQuicStreamState</a></code>
               changes.</p>
             </dd>
-            <dt><code>readBufferedAmount</code> of type <span class="idlAttrType"><a>unsigned
+            <dt><code>readableAmount</code> of type <span class="idlAttrType"><a>unsigned
             long</a></span>, readonly</dt>
             <dd>
-              <p>The <dfn id="dom-quicstream-readbufferedamount"><code>readBufferedAmount</code></dfn>
+              <p>The <dfn id="dom-quicstream-readableamount"><code>readableAmount</code></dfn>
               attribute represents the number of bytes buffered for access by
               <code>readInto</code> but that, as of the last time the event loop
               started executing a task, had not yet been read. This does not include
               framing overhead incurred by the protocol, or buffers associated with
               the network hardware. On getting, it <em class="rfc2119" title="MUST">MUST</em>
               return the value of the <code><a>RTCQuicStream</a></code>'s
-              <a>[[\ReadBufferedAmount]]</a> internal slot.
+              <a>[[\ReadableAmount]]</a> internal slot.
               If the <code><a>RTCQuicStream</a></code> is in the <code>closed</code> state,
               this attribute's value will only decrease with each call to the
               <code>readInto</code> method (the attribute does not reset to zero once the
               <code><a>RTCQuicStream</a></code> closes).</p>

```

Comment:
Didn't we decide that when the RTCQuicStream closes that the read and write buffers are cleared and [[Readable]] and [[Writable]] transition to false? Currently, the spec already states that [[Readable]] and [[Writable]] should transition to false. In this case the following should be removed:

> (the attribute does not reset to zero once the <code><a>RTCQuicStream</a></code> closes)"